low-level/sspl-ll.spec

```diff
@@ -140,6 +140,14 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
 [ -f $SSPL_DIR/sspl_init ] ||
     ln -s $SSPL_DIR/bin/sspl_provisioner_init $SSPL_DIR/sspl_init
 
+# Creating softlink under site-packages to use sspl module easier way.
+PYTHON_BASE_DIR=/usr/lib/python3.6/site-packages/cortx/sspl
+[ -d "${SSPL_DIR}" ] && {
+    mkdir -p $PYTHON_BASE_DIR
+    ln -sf $SSPL_DIR/bin $PYTHON_BASE_DIR/bin
```

Comment:
You can bypass bin, i.e. 
ln -s {sspl}/bin/xxx site-packages/cortx/sspl/xxx

Reply:
Under sspl/bin directory we have 25(xxxx) files. 
So at this time, keeping this symlink for bin would just be fine.

Reply:
agree, we need to find better approach instead of putting 25+ files symlinks in site-packages/cortx/sspl/
So current one is fine.